common/content/statusline.js

```diff
@@ -130,8 +130,7 @@ const StatusLine = Module("statusline", {
         if ( typeof this.setVisibility.currVisibility == 'undefined' ) {
             this.setVisibility.currVisibility = true;
             this.setVisibility.prevVisibility = true;
-            this.setVisibility.contentSeparator
-                = highlight.get('ContentSeparator').value;
+            this.setVisibility.passiveVisibility = false;
```

Comment:
It sounds like we should just change update the value of `currVisbility`. I don't think it would be good to add yet another flag.